types/mercadopago/resources/payment.d.ts

```diff
@@ -22,6 +23,8 @@ export interface MercadoPagoPayment {
 
   get(id: number, configuration?: DefaultConfigurationOmitQs, callback?: CallbackFunction): Promise<PaymentGetResponse>;
 
+  capturePartial(payload: CapturePartialPayload, configuration?: DefaultConfigurationOmitQs, callback?: CallbackFunction): Promise<PaymentCreateResponse>;
```

Comment:
The response is indeed the same as the `create` method but in reality we should change it to `PaymentUpdateResponse` and on line `export type PaymentUpdateResponse = MercadoPagoResponse<ExecOptions<DefaultConfigurationOmitQs, UpdatePaymentPayload>>;` you should change it to `export type PaymentUpdateResponse = MercadoPagoResponse<ExecOptions<DefaultConfigurationOmitQs, PaymentCreateResponse>>;`. This is because the `capturePartial` calls `update` method under the hood so the response should be the same as update. Also the type I created `PaymentUpdateResponse` is wrong because the response is the same as the `PaymentCreateResponse`. So it would be like that:
```
...
export type PaymentUpdateResponse = MercadoPagoResponse<ExecOptions<DefaultConfigurationOmitQs, CreatePaymentPayload>>;
...
capturePartial(...): Promise<PaymentUpdateResponse>;
....
```